runtime/bcutil/cfreader.c

```diff
@@ -2434,7 +2440,8 @@ j9bcutil_readClassFileBytes(J9PortLibrary *portLib,
 		classfile->accessFlags |= CFR_ACC_ABSTRACT;
 	}
 
-	if (J9_ARE_ALL_BITS_SET(classfile->accessFlags, CFR_ACC_MODULE)) {
+	if ((vmVersionShifted >= BCT_Java9MajorVersionShifted)
```

Comment:
This check was throwing an error in the Java 8 builds. The change to CFR_CLASS_ACCESS_MASK was also causing failures.